### ISSUE TITLE
Force a token refresh?

### ISSUE THREAD

**Issue by scalemaildev** [danielroe/nuxt-turnstile#84]
### 🆒 Your use case

Currently my account creation flow has the turnstile token validated, then some account creation stuff runs. If the account fails to create, the user is notified of why, and asked to fix the creation form.

However, the turnstile token is now invalid. So the user has to either refresh the page or wait for the token to regenerate.

### 🆕 The solution you'd like

Is there any way to manually force a token refresh, so that I can refresh it whenever my submission form gets kicked back?

**Comment by danielroe** [comment]
Yes, there is - implemented via https://github.com/danielroe/nuxt-turnstile/commit/08085242aeb9a58335646e8c83e48605d502a78c - you should see an example usage in the playground.

A PR to the docs to add some info would be welcomed 🙏

**Comment by scalemaildev** [comment]
I could contribute to your docs. 

However, the function only appears to be exposed to a button click for some reason.

I.E I can refresh the token with a button that has "turnstile.reset()" as its clickable event, but I when calling it programmatically I get "turnstile.reset is not a function."

**Comment by danielroe** [comment]
it would be `turnstile.value.reset`

**Comment by scalemaildev** [comment]
Ok I opened a pr with some updated docs.

**Comment by luizzappa** [comment]
This reset function is still not very clear to me. The generation of a new token is not synchronous, and it is also not a Promise, how can I wait for the generation of this token to send it to the API when submitting the forms?
